### PR TITLE
Adds JWT_SECRET and V1SCHEMA env variables

### PR DESCRIPTION
**Pull request by diogob**
Just so we can set these parameters up when deploying to heroku or dokku.

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -74,7 +74,7 @@ cp -R $CACHE_DIR/$BIN $BUILD_DIR
 
 echo "-----> Creating Procfile"
 cat <<EOF > $BUILD_DIR/Procfile
-web: ./${BIN} -p \$PORT -s -d \$DB_NAME -U \$AUTH_ROLE --db-pass \$AUTH_PASS -a \$ANONYMOUS_ROLE --db-host \$DB_HOST -P \$DB_PORT --db-pool \$DB_POOL
+web: ./${BIN} -p \$PORT -s -d \$DB_NAME -U \$AUTH_ROLE --db-pass \$AUTH_PASS -a \$ANONYMOUS_ROLE --db-host \$DB_HOST -P \$DB_PORT --db-pool \$DB_POOL --jwt-secret \${JWT_SECRET=secret} --v1schema \${V1SCHEMA=1}
```
Comment (begriffs) [inline bin/compile]: Should it be `${JWT_SECRET:-secret}` rather than `${JWT_SECRET=secret}`?

Comment (diogob) [inline bin/compile]: That's right :astonished: 
My mistake

**Comment by timwis** [comment]
Interesting... by putting a default value in the command line, it's not necessary to add it to the [deploy to heroku env vars](https://github.com/begriffs/postgrest/blob/master/app.json). These 2 would be the only 2 env vars like that.

**Comment by begriffs** [comment]
Do you have an opinion which is better? (Defaults in the buildpack vs in app.json)

**Comment by timwis** [comment]
If it's just in the buildpack, you'd have to dig around to find that it's
an env var at all. If it's in the app.json it will show up when you click
deploy to heroku. Including defaults there will hopefully decrease the
"config overwhelm" factor. I'd suggest app.json for that reason.

**Comment by diogob** [comment]
True, maybe it's better to keep it consistent and add everything in the app.json. I'll update the file.

**Comment by begriffs** [comment]
Looks good.

**Pull request merged**